Header header
int8 obstacle_ID
float32 x
float32 y
float32 Kr
float32 gamma
float32 influence
